code.js

```diff
@@ -0,0 +1,76 @@
+//Kruskal's Algorithm
+function kruskal(graph) {
+
+    //Start with an empty tree T
+    let tree = [];
+    for (let i = 0; i < graph.length; i++) {
+        tree[i] = [];
+    }
+    //Build a priority queue
+    let queue = [];
+    for (let i = 0; i < graph.length; i++) {
+        for (let j = 0; j < graph[i].length; j++) {
+            queue.push([i, j, graph[i][j][1]]);  //[row, col, cost]
+        }
+    }
+    mergesort(queue);
+
+    //Add the minimum weight edge to T unless it forms a cycle
+    for (edge of queue) {
+        tree[edge[0]].push(graph[edge[0]][edge[1]]);
+        if (hasCycle(tree)) {
+            tree[edge[0]].splice(tree[edge[0]].length - 1, 1);
+        }
+    }
+    return tree;
+}
+
+//Sorting function adapted from class notes
```

Comment:
Uhhh... you can just use `.sort()`. No need to implement your own sorting function.

Reply:
Oh, I forgot about that.  Do I have to use the built-in sort?  Is it more efficient than what I've already written?

Reply:
You don't have to, but it's certainly more efficient.